<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>button_login</class>
 <widget class="QDialog" name="button_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(95, 87, 87);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignRight|Qt::AlignTop">
    <widget class="QFrame" name="frame_7">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="home_screen_logout_push_button">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>161</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item alignment="Qt::AlignHCenter|Qt::AlignTop">
       <widget class="QLabel" name="home_screen_welcome_label">
        <property name="styleSheet">
         <string notr="true">color: rgb(247, 247, 247); 
font: 26pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Welcome</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="home_screen_all_pqs_push_button">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>All PQs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="home_screen_unallocated_pqs_push_button">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Unallocated PQs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="home_screen_allocated_pqs_push_button">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Allocated PQs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="frame_5">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="home_screen_useful_documents_push_button">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Useful Documents</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QFrame" name="frame_6">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="home_screen_settings_push_button">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>200</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(235, 235, 235);
font: 16pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
